spec/emittr_spec.rb

```diff
@@ -103,6 +103,85 @@
     end
   end
 
+  describe '#on_any' do
+    let(:block)     { Proc.new {} }
+    let(:any_block) { Proc.new {} }
+
+    context 'when #emit is called' do
+      it 'calls #on_any blocks' do
+        emitter.on_any &any_block
+
+        emitter.on :first_any_test, &block
+        emitter.on :second_any_test, &block
+
+        expect(any_block).to receive(:call).exactly(4).times
+
+        2.times { emitter.emit :first_any_test }
+        2.times { emitter.emit :second_any_test }
+      end
+    end
+  end
+
+  describe '#off_any' do
+    let(:block) { Proc.new {} }
+
+    before do
+      emitter.on(:off_any_test) { Proc.new {} }
+
+      emitter.on_any &block
+      emitter.off_any &block
+    end
+
+    it 'removes listener from #any list' do
+      emitter.emit :off_any_test
+      listeners = emitter.send(:listeners)
+
+      expect(listeners[:any]).to be_empty
+    end
+
+    it "doesn't call block" do
+      expect(block).not_to receive(:call)
```

Comment:
We should expect for a call, then use `off_any`, then test if will not be called again. Since we're describing `#off_any` here, the call for this method should be inside the tests, not inside the `before`.